e2e/test/scenarios/permissions/application-permissions.cy.spec.js

```diff
@@ -84,7 +83,12 @@ describeEE("scenarios > admin > permissions > application", () => {
 
         visitQuestion(ORDERS_QUESTION_ID);
         tableInteractive().should("be.visible");
-        sharingMenuButton().should("be.disabled");
+        openSharingMenu();
```

Comment:
In the upcoming PR, nothing on the sharing button will change anymore. Only the embed modal (that shows 3 embed options) would be changed.

The sharing button will always be clickable and show at least the embed option for both questions and dashboards. This will remain unchanged.

Reply:
That makes sense - thanks!